app/assets/javascripts/app/core/me.js

```diff
@@ -0,0 +1,37 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.core')
+    .service('Me', Me);
+
+  function Me($http, $q) {
+    var me;
+    var mePromise;
+
+    function find() {
+      mePromise = $http.get('/me')
+        .then(set);
+      return mePromise;
+    }
+
+    function set(response) {
+      me = response.data;
+      return me;
```

Comment:
When using a data fetching service like this, we try to avoid retrieving and manipulating the actual data (state) wherever that service is being used/called. We strive for a more functional approach, through methods that manipulate that state for you.
So, returning the state here could be considered an anti-pattern. If you don't allow it from the beginning it'll be easier not to go that stateful approach.